examples/phaser/ADAR_pyadi_functions.py

```diff
@@ -0,0 +1,164 @@
+    array.elements.get(4).rx_gain = int(Gain4 * 127 / 100 * gcal[3])
+    array.elements.get(4).rx_attenuator = not bool(Gain4)
+    array.elements.get(5).rx_gain = int(Gain5 * 127 / 100 * gcal[4])
+    array.elements.get(5).rx_attenuator = not bool(Gain5)
+    array.elements.get(6).rx_gain = int(Gain6 * 127 / 100 * gcal[5])
+    array.elements.get(6).rx_attenuator = not bool(Gain6)
+    array.elements.get(7).rx_gain = int(Gain7 * 127 / 100 * gcal[6])
+    array.elements.get(7).rx_attenuator = not bool(Gain7)
+    array.elements.get(8).rx_gain = int(Gain8 * 127 / 100 * gcal[7])
+    array.elements.get(8).rx_attenuator = not bool(Gain8)
+    array.latch_rx_settings()
+
+
+# MWT: Consider combining phasex into an array.
+def ADAR_set_Phase(
+    array,
+    PhDelta,
+    phase_step_size,
+    phase1,
+    phase2,
+    phase3,
+    phase4,
+    phase5,
+    phase6,
+    phase7,
+    phase8,
+):
+    step_size = phase_step_size  # 2.8125
+    array.elements.get(1).rx_phase = (
+        (np.rint(PhDelta * 0 / step_size) * step_size) + phase1 + pcal[0]
+    ) % 360
+    array.elements.get(2).rx_phase = (
+        (np.rint(PhDelta * 1 / step_size) * step_size) + phase2 + pcal[1]
+    ) % 360
+    array.elements.get(3).rx_phase = (
+        (np.rint(PhDelta * 2 / step_size) * step_size) + phase3 + pcal[2]
+    ) % 360
+    array.elements.get(4).rx_phase = (
+        (np.rint(PhDelta * 3 / step_size) * step_size) + phase4 + pcal[3]
+    ) % 360
+    array.elements.get(5).rx_phase = (
+        (np.rint(PhDelta * 4 / step_size) * step_size) + phase5 + pcal[4]
+    ) % 360
+    array.elements.get(6).rx_phase = (
+        (np.rint(PhDelta * 5 / step_size) * step_size) + phase6 + pcal[5]
+    ) % 360
+    array.elements.get(7).rx_phase = (
+        (np.rint(PhDelta * 6 / step_size) * step_size) + phase7 + pcal[6]
+    ) % 360
+    array.elements.get(8).rx_phase = (
+        (np.rint(PhDelta * 7 / step_size) * step_size) + phase8 + pcal[7]
+    ) % 360
+    array.latch_rx_settings()
+
+
+def load_gain_cal(filename="gain_cal_val.pkl"):
+    """ Load gain calibrated value, if not calibrated set all channel gain to maximum.
+        parameters:
+            filename: type=string
+                      Provide path of gain calibration file
+    """
+    try:
+        with open(filename, "rb") as file1:
+            return pickle.load(file1)  # Load gain cal values
+    except:
+        print("file not found, loading default (all gain at maximum)")
+        return [1.0] * 8  # .append(0x7F)
+
+
+def load_phase_cal(filename="phase_cal_val.pkl"):
+    """ Load phase calibrated value, if not calibrated set all channel phase correction to 0.
+        parameters:
+            filename: type=string
+                      Provide path of phase calibration file
+    """
+
+    try:
+        with open(filename, "rb") as file:
+            return pickle.load(file)  # Load gain cal values
+    except:
+        print("file not found, loading default (no phase shift)")
```

Comment:
Need explicit exception and not generic one

Reply:
Done.